pkgs/development/python-modules/batchgeneratorsv2/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  batchgenerators,
+  fft-conv-pytorch,
+  numpy,
+  torch,
+}:
+
+buildPythonPackage rec {
+  pname = "batchgeneratorsv2";
+  version = "0.3.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "MIC-DKFZ";
+    repo = "batchgeneratorsv2";
+    tag = "v${version}";
+    hash = "sha256-UoEYSuDD+6ResA0luCZ3LxKGp9H9Mkcnvok/O673ITk=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  dependencies = [
+    batchgenerators
+    fft-conv-pytorch
+    numpy
+    torch
+  ];
+
+  pythonImportsCheck = [
+    "batchgeneratorsv2"
+  ];
+
+  meta = {
+    description = "";
```

```suggestion
    description = "";
```
please add description